teuthology/task/selinux.py

```diff
@@ -20,6 +20,15 @@ class SELinux(Task):
     must first be enabled and the filesystem must have been labeled.
 
     On teardown, also checks the audit log for any denials.
+    By default the test will fail for any denials seen in audit.log, for the
```

Comment:
Yeah, I think it's just a matter of improving the wording. Just please document the defaults and note that they are always used regardless of if se-whitelist is passed in via a config.